grakn-graph/src/main/java/ai/grakn/graph/internal/Cache.java

```diff
@@ -74,6 +74,10 @@ public V get(){
         return cachedValue.get();
     }
 
+    V copyShared(){
+        return sharedValue.get();
+    }
```

Comment:
We should add a comment/javadoc indicating this method is intended to be overridden.